examples/tree_2d_dgsem/elixir_shallowwater_wall.jl

```diff
@@ -0,0 +1,83 @@
+using OrdinaryDiffEq
+using Trixi
+
+###############################################################################
+# Semidiscretization of the shallow water equations
+
+equations = ShallowWaterEquations2D(gravity_constant = 9.81, H0 = 3.25)
+
+# An initial condition with a perturbation in the waterheight to test boundary_condition_slip_wall
+function initial_condition_well_balancedness(x, t, equations::ShallowWaterEquations2D)
```

Comment:
This is a weird name for the initial condition because it is not a well-balancedness testcase. Maybe something like `initial_condition_perturbation`?

Reply:
Yes, I forgot to change this. I changed the name to `initial_condition_perturbation` in 7d03b4a65dfa508d5367916abd9a7497d1fa6407